docs/source/algorithms/SetSample-v1.rst

```diff
@@ -71,14 +71,20 @@ For defining the full shape a key called ``Shape`` specifying the desired shape
 expected along with additional keys specifying the values (all values are assumed to
```

Comment:
While at it, you could perhaps fix the typo in the first sentence of the **Environment** subsection as well: 'enviorment' -> 'environment'.